valeriano-manassero/trino/templates/deployment-coordinator.yaml

```
@@ -88,10 +96,65 @@ spec:
           configMap:
             name: {{ .configMapName }}
         {{- end }}
-      {{- if .Values.config.coordinator.initContainers }}
+      {{- if or .Values.config.coordinator.initContainers (include "trino.tlsEncryption" . ) }}
       initContainers:
-      {{-  tpl (toYaml .Values.config.coordinator.initContainers) . | nindent 6 }}
+      {{- if (include "trino.tlsEncryption" . ) }}
+        - name: init-certs
+          image: {{ .Values.initKeystore.image.repository }}:{{ .Values.initKeystore.image.tag }}
+          imagePullPolicy: {{ .Values.initKeystore.image.pullPolicy }}
+          {{- with .Values.image.securityContext }}
+          securityContext:
+            runAsUser: {{ .runAsUser }}
+            runAsGroup: {{ .runAsGroup }}
+          {{- end }}
+          command: [ /bin/bash ]
```

Although JSON syntax is valid YAML, I am partial to YAML syntax...
```suggestion
          command:
            - /bin/bash
```